### ISSUE TITLE
Popup keyboard navigation bug when dismissing popup

### ISSUE THREAD

**Issue by prushforth** [Maps4HTML/Web-Map-Custom-Element#681]
When you dismiss a popup with "Escape" key or by tabbing to and activating the "X" affordance, the focus should fall back on the feature that is being interrogated, and a tooltip should be displayed / read.  While focus usually goes to the feature, no tooltip is displayed.
